docs/AutomationReference.md

```diff
@@ -1,5 +1,7 @@
 ## Axe.Windows - automation reference
 
+Note: The following documentation describes the supported API provided by Axe.Windows. There are other, public interfaces and classes in various assemblies throughout the package; however, please be aware interfaces and classes not described in this document are not supported and may change at any time.
```

Comment:
I think this text is good. I do wonder whether we should put it at a 'higher' level in the repo such as the root .README file. That way it will be hard to miss regardless of what part of the docs / code one explores?

Reply:
I looked at the readme, but I didn't feel like I found a good fit there. Suggestions?